GliaWidgets/SecureConversations/SecureConversations.ActitivityIndicator.swift

```diff
@@ -0,0 +1,202 @@
+import UIKit
```

Comment:
File still has a typo because it's called Actitivity

Reply:
Sigh a8521400bb50c13824bef4a83831901cd91f625d.